### ISSUE TITLE
Scrollview horizontal pagination

### ISSUE THREAD

**Issue by sebakerckhof** [Famous/famous-angular#312]
This codePen is adapted from the famous-angular docs: http://codepen.io/anon/pen/EaWgXb . Adapted from: https://famo.us/integrations/angular/docs/unstable/api/directive/faScrollView/index.html

I want to have to pages where users can swipe between. I test this in chrome with emulate touchscreen, and while I can drag the surfaces, it never snaps to the second page. When I swipe faster, everything disappears and it just becomes plain white...

**Comment by jordanpapaleo** [comment]
Hi @sebakerckhof - 

So in your demo you are using Famo.us 0.3.4.  The example in the docs uses Famo.us 0.2.2.  Using [this](https://code.famo.us/famous/global/0.2.2/famous.min.js) version of famous does resolve your issue.  Are you able to use this version?  If so, for now, I would have to recommend using 0.2 if you are going to work off of the examples as they have not been ported to 0.3.  I will bring up the need for updating all of the examples in the docs.

Thanks,

Jordan

**Comment by jordanpapaleo** [comment]
UPDATE: 

Hi @sebakerckhof -

So I just ran into this and spent some time working it out.  I am now able to accomplish this with the current version of famous-angular.  There is still a version issue but this works.  So you cant have undefined in your modifier for size.  What I did was put a function on scope that returns the window.innerWidth.  Check out this [codepen](http://codepen.io/anon/pen/xbrpjd)  I hope that works for you.

Thanks,

Jordan
